contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2231,6 +2256,15 @@ class tsqlBuilder : public tsqlCommonMutator
 	{
 		handleOrderByOffsetFetch(ctx);
 	}
+
+	void exitTRIM(TSqlParser::TRIMContext *ctx) override
+	{
+		if (ctx->trim_from())
+		{	
+			rewritten_query_fragment.emplace(std::make_pair(ctx->trim_from()->start->getStartIndex(), std::make_pair(::getFullText(ctx->trim_from()), "  , ")));
+		}
+		rewritten_query_fragment.emplace(std::make_pair(ctx->start->getStartIndex(), std::make_pair(::getFullText(ctx->TRIM()), "sys.TRIM")));
```

Comment:
**Code Repetition**, let's move this code section to a common function and we can simply invoke it from here and line 929.

Reply:
Done